### DISCUSSION TITLE
How to connect with only the link cable and no internet (Oculus Quest 2)

### DISCUSSION THREAD

**Discussion by samanake** [alvr-org/ALVR#634] [Q&A]
I've seen some posts saying it's possible to connect to ALVR without internet at all.
For context, I have the link cable connected to my PC that is connected to my 2.4GHz router through ethernet.
I was able to create a Trusted entity and play, but playing SteamVR games through a 2.4GHz internet connection isn't feasible.
But why do I need the internet connection if I have the link cable? I've tried disabling wifi on the Quest to get it to force a USB connection but it still tries to connect through the internet.

**Comment by HoomanDiscord** [comment]
You misunderstood what they meant, they meant that you don't need an active internet connection but only need a connection between your quest and your computer

**Reply by samanake** [reply]
Mm. I see. And I just stupidly discovered that I can just turn on SteamVR through the Oculus Link menu.

**Reply by sctanf** [reply]
The headset needs to be connected to wifi, but that is the only requirement, since the actual connection will be through USB.
